Gemfile

```diff
@@ -7,3 +7,5 @@ group :test do
 end
 
 gem 'travis', '~> 1.8', '>= 1.8.8'
+
+gem 'sinatra'
```

Comment:
As per the Zagaku this morning, I'd recommend adding the version number here. If you go into your terminal and type `gem list`, I think it should show you which version of Sinatra is installed. And then you can add that to your Gemfile. So if I were to clone this code and did a `bundle install`, then I'd get the correct version of Sinatra.